app/controllers/course/announcements_controller.rb

```diff
@@ -2,7 +2,8 @@ class Course::AnnouncementsController < Course::ModuleController
   load_and_authorize_resource :announcement, through: :course, class: Course::Announcement.name
 
   def index #:nodoc:
-    @announcements = @announcements.includes(:creator).sorted_by_date.page params[:page]
+    @announcements = @announcements.includes(:creator).sorted_by_sticky.sorted_by_date
+      .page params[:page]
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.